### DISCUSSION TITLE
Ui framework

### DISCUSSION THREAD

**Discussion by parthjain0** [infinitered/ignite#2655] [General]
Hey guys do you have any recommendations for the rn ui libraries we should use with this setup like paper or nativebase? Thanks for the template anyway.

**Comment by frankcalise** [comment]
Hi @parthjain0 - it really depends on the project and what you like, but I know some IR members have used UI Kitten before with good success!

React Native Paper is another popular library.

**Reply by parthjain0** [reply]
thanks @frankcalise  will check it out 😃
